packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -49,6 +49,35 @@ to our CDK project directory. This is especially important when we want to share
 this construct through a library. Different programming languages will have
 different techniques for bundling resources into libraries.
 
+When using `fromAsset` or `fromInline`, you can obtain the hash of source
+through the `function.codeHash` property. This property will return `undefined`
+if the code hash cannot be calculated during synthesis (e.g. when using code
+from an S3 bucket).
+
+### Versions and Aliases
```

Comment:
Include a reference to [`latestVersion`](https://github.com/aws/aws-cdk/blob/746ba3247a86a0cca60a1bb7897dd12848d904e9/packages/%40aws-cdk/aws-lambda/lib/function-base.ts#L237-L240) and the clarity on the differences and when to use either.